.travis.yml

```diff
@@ -16,7 +16,7 @@ before_script:
     - (cd EpsilonLights && CubeMX2Makefile .)
```

Comment:
@islamrahat735 this line is probably what's shafting you rn. Delete it and see if your build passes

Reply:
It worked!